.github/workflows/validate-devfiles.yaml

```diff
@@ -28,81 +28,133 @@ concurrency:
 
       - name: Check that containers components are non terminating
         run: tests/check_non_terminating.sh
 
-  odov3:
-    name: with odo v3
+  odov2:
+    name: with odo v2
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v1
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
       - name: Setup Minikube
         uses: manusa/actions-setup-minikube@v2.7.0
         with:
           minikube version: ${{ env.MINIKUBE_VERSION }}
           kubernetes version: ${{ env.KUBERNETES_VERSION }}
-          driver: 'docker'
+          driver: "docker"
           github token: ${{ secrets.GITHUB_TOKEN }}
-          start args: '--addons=ingress'
+          start args: "--addons=ingress"
 
-      - name: Install Go
-        uses: actions/setup-go@v3
+      - name: Install odo v2
+        uses: redhat-actions/openshift-tools-installer@v1
         with:
-          go-version: '1.19'
-      - name: Install odo v3
-        # Uses odo from main branch.
-        # Once odo v3.0.0 is officially released this should be updated to use redhat-actions/openshift-tools-installer@v1
-        run: |
-          git clone https://github.com/redhat-developer/odo /tmp/odo
-          cd /tmp/odo
-          make install
-      - name: 'Install Ginkgo'
-        run: go install -mod=mod github.com/onsi/ginkgo/v2/ginkgo@v2.1.4
+          odo: "2.5.1"
+
       - name: Check odo version
         run: odo version
-      - name: Check the devfile stacks with odo v3
-        run: tests/check_odov3.sh
 
-  odov2:
-    name: with odo v2
+      - name: Test all stacks if on a schedule
+        if: ${{ github.event_name == 'schedule' }}
+        run: echo "TEST_ALL_STACKS=true" >> $GITHUB_ENV
+
+      - name: Check the devfile stacks with odo v2
+        run: tests/check_odov2.sh odo
+
+  odov3:
+    name: with odo v3
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v1
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
       - name: Setup Minikube
         uses: manusa/actions-setup-minikube@v2.7.0
         with:
           minikube version: ${{ env.MINIKUBE_VERSION }}
           kubernetes version: ${{ env.KUBERNETES_VERSION }}
-          driver: 'docker'
+          driver: "docker"
           github token: ${{ secrets.GITHUB_TOKEN }}
-          start args: '--addons=ingress'
+          start args: "--addons=ingress"
 
-      - name: Install odo v2
-        uses: redhat-actions/openshift-tools-installer@v1
+      - name: Install Go
+        uses: actions/setup-go@v3
         with:
-          odo: '2.5.1'
+          go-version: "1.19"
+
+      - name: Install odo v3
+        # Uses odo from main branch.
+        # Once odo v3.0.0 is officially released this should be updated to use redhat-actions/openshift-tools-installer@v1
+        run: |
+          git clone https://github.com/redhat-developer/odo /tmp/odo
```

Comment:
Can you update this to use `redhat-actions/openshift-tools-installer@v1` now that Odo v3 is out?

Reply:
sure, this will be in the next commit